### PR TITLE
AT-13-File Uploader

### PR DESCRIPTION
**Pull request by grantwhiteman**
Created front end for file uploader, takes input of a file and shows success or errors.
Tested with Cypress in API spec

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,8 +40,12 @@
     ]
   },
   "devDependencies": {
+<<<<<<< HEAD
```
Comment (dearshrewdwit) [inline package.json:43]: 👀

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (benclayton98) [reply package.json]: we've made this change but the jenkins tests on github are still failing, are they supposed to be?

Comment (dearshrewdwit) [reply package.json]: You left a bunch of conflict remnants in this file!

Comment (benclayton98) [reply package.json]: Oh my bad! Think that was from a rebase that didn't work as intended

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
CI is currently failing

```sh
+ npm run build
> atlas-frontend@0.1.0 build
> react-scripts build
Creating an optimized production build...
Treating warnings as errors because process.env.CI = true.
Most CI servers set it automatically.

Failed to compile.

src/components/Upload/Response.js
  Line 4:5:  Useless constructor  no-useless-constructor
script returned exit code 1
```

To fix - change the stateful component into a stateless component (class -> functional)

**Review by b7rch4 — DISMISSED** [review]

**Comment by dearshrewdwit** [comment]
Very tempted to ask you to force squash your commits - this should be a habit!

28 commits is a large number for any 1 feature.

**Review by dearshrewdwit — APPROVED** [review]

**Review by Neokrieger — APPROVED** [review]
Very good

**Pull request merged**